src/game_strings.cpp

```diff
@@ -183,13 +183,17 @@ StringView Game_Strings::ToFile(Str_Params params, std::string filename, int enc
 	}
 
 	// Maniacs forces the File in Text/ folder with .txt extension
-	// TODO: Maybe relax this?
-	filename = "Text/" + filename + ".txt";
+	filename = "Text/" + filename;
+	// If "*" is at the end of filename, the file path and extension are relaxed - FEATURE EXCLUSIVE TO EASYRPG
+	if (filename.back() == '*') filename.pop_back();
```

Comment:
does this really work as intended? This only relaxes the extension, not the path (still prefixed with ``Text/``.

Reply:
Oops, you are correct. 
I abandonned the relaxing path idea. 
It would just confuse people, since they can do `/../`to nav out of the text/ folder.